db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1605,6 +1641,7 @@ Status DBImpl::FlushMemTable(ColumnFamilyData* cfd,
         nonmem_write_thread_.ExitUnbatched(&nonmem_w);
       }
     }
+    }
```

Comment:
Format, not adding indent here.

Reply:
Yes, I didn't add indent here intentionally to avoid change the whole block.

Reply:
You can add a comment here.

```suggestion
    } // if (!pegasus_data_ || !cfd->mem()->IsEmpty())
```